client/boot/index.js

```diff
@@ -174,7 +174,7 @@ function renderLayout( reduxStore ) {
 	let props = { focus: layoutFocus };
 
 	if ( user.get() ) {
-		Object.assign( props, {}, { user, sites, nuxWelcome, translatorInvitation } );
+		Object.assign( props, { user, sites, nuxWelcome, translatorInvitation } );
```

Comment:
Cleanup related to [this](https://github.com/Automattic/wp-calypso/pull/5748#discussion_r66670661)